pkg/core/system.go

```diff
@@ -22,18 +22,18 @@ import (
 	"errors"
 	"fmt"
 	"io/ioutil"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"net/http"
 	"net/url"
 	"os"
 	"strings"
 	"time"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
 const (
 	istioNamespace  = "istio-system"
-	istioCrds       = "https://storage.googleapis.com/riff-releases/istio/istio-1.0.0-riff-crds.yaml"
-	istioRelease    = "https://storage.googleapis.com/riff-releases/istio/istio-1.0.0-riff-main.yaml"
+	istioRelease    = "https://github.com/knative/serving/raw/master/third_party/istio-1.0.1/istio-lean.yaml"
```

Comment:
not sure we should point to a git file that doesn't have a tag or isn't from the release bucket, it could be gone when they decide to use istio 1.0.2 instead

Reply:
and I assume we'd be better off using `istio.yaml` instead of the lean one